set-web/src/main/java/eu/europeana/set/web/service/controller/jsonld/WebUserSetRest.java

```diff
@@ -299,6 +302,8 @@ private void checkVisibilityForRetrieve(UserSet userSet, HttpServletRequest requ
 	    // if false respond with HTTP 400
 	    getUserSetService().validateWebUserSet(newUserSet);
 
+	    getUserSetService().validateFavoriteUserSet(newUserSet, existingUserSet);
```

Comment:
please remove and integrate in validateWebUserSet

Reply:
I can do it if you disagree with my previous comment

Reply:
please remove and see the previous indications

Reply:
removed according to previous indications